code/scripts/deploy_stage.sh

```diff
@@ -137,8 +139,7 @@ copy_images() {
     rm -rf "$CUR_DIR"deploy/images
   fi
   mkdir -p "$CUR_DIR"deploy/images
-  cp -r "$CUR_DIR"drasil-website/WebInfo/images/* "$CUR_DIR"deploy/images
-  
+  cp -r "$CUR_DIR/drasil-website/WebInfo/images/" "$CUR_DIR/deploy/images"
```

Comment:
```suggestion
  cp -r "$CUR_DIR/drasil-website/WebInfo/images/" "$CUR_DIR/deploy"
```

Currently, a double images folder is being formed within deploy. 

![image](https://github.com/JacquesCarette/Drasil/assets/110783820/bee615f1-bfa0-46c1-8948-973a1b1b2723)


Removing the `/images` part would prevent this from happening.